front/src/applications/stdcm/utils/checkStdcmConfigErrors.ts

```diff
@@ -25,6 +25,10 @@ const checkStdcmConfigErrors = (
     return { errorType: StdcmConfigErrorTypes.PATHFINDING_FAILED };
   }
 
+  if (origin.name === destination.name && origin.ch === destination.ch) {
```

Comment:
Would be more reliable to check the `uic` here instead of `name`: `name` is a human-readable string which may not be unique.

Reply:
Looks good! Unfortunately we still have to write these `'uic' in …` checks, hopefully this can go away once Clara's refactorings are merged.